console/billing/concepts.mdx

```diff
@@ -5,16 +5,143 @@ meta:
+
+</Concept>
+
+<Concept opened>
+
+## Free Trial
+
+The Free Trial provides users access to specific products at no cost. However, it is available only for a limited duration and usage quantity.
+
+</Concept>
+
+<Concept opened>
+
+## IAM billing permission
+
+Each group is linked to a permission set. There are two permission sets linked to billing:
+
+Billing read only: List and read access to billing information. This allows the user to view consumption, invoices, discounts, budget alerts and payment methods.
+Billing Manager: Full access to billing management. This allows the user to list, read and edit billing contact information, payment information, billing alerts and invoices.
+
+</Concept>
+
+<Concept opened>
+
+## Pay as you go 
+
+Public cloud services follow a 'pay as you go' model, ensuring you pay only for what you use. Some products offer plan options. Your monthly invoice, calculated at the end of the month, is based on exact resource usage and may include additional invoices for upfront payments during the month.
+
+</Concept>
+
+<Concept opened>
+
+## Payment methods
+ 
+The payment methods registered on your account are used to pay your invoice. You can set one of your payment methods as the default and manage your different payment methods from the console.
+
+</Concept>
+
+<Concept opened>
+
+## Project
+
+A Project is a grouping of Scaleway [resources](#resource). Each Scaleway Organization comes with a default Project, and you can create new Projects if necessary. Projects are cross-regional, meaning resources located in different [regions](/compute/instances/concepts/#region) can be grouped into one single Project. When grouping resources into different Projects, you can use [IAM](/identity-and-access-management/iam/concepts/#iam) to define custom access rights for each Project.
+
+</Concept>
+
+<Concept opened>
+
+## Pro-rata billing
+
+You are billed pro-rata on Dedibox and Web Hosting services for the initial month of a subscription. This means the charge is adjusted to reflect the portion of the first month you used the service. Starting from the second month and onwards, you are billed in full at the beginning of each month, covering the entire upcoming billing period in advance. This billing model ensures that you pay for the service based on your actual usage for the first month and then in advance for subsequent months.
+
+</Concept>
+
+<Concept opened>
+
+## Support plan 
+
+Scaleway provides four different types of [support plans](https://console.scaleway.com/support/plans): Basic, Bronze, Silver and Gold. Your support plan determines the level of service and dedicated assistance you have access to, and the guaranteed response time of your support requests. You can [configure your support plan in the console](/console/account/how-to/configure-support-plans/).
+
+</Concept>
+
 <Concept opened>
 
 ## Voucher
 
-</Concept>
```

Comment:
to be removed

Reply:
It's already removed (that's why the line is red) :)